client/zone/js-modules/map-drawer.js

```diff
@@ -21,9 +21,9 @@ export function MapDrawer() {
 
     /**
      *
-     * @param {Array<Array<number>>} mapData - the data containing the map image (array of pixel offsets and colors)
+     * @param {Array<object>} layers - the data containing the map image (array of pixel offsets)
```

Comment:
We should use fixed types so that type checking works properly.

Reply:
I agree, however that should happen outside of this PR since it's already insanely huge